rate-limit-cloud-controller-api.html.md.erb

```diff
@@ -6,6 +6,7 @@ owner: CAPI
 This topic describes rate limit information that is returned by the Cloud Controller API (CAPI).
 
 For information about how to configure rate limits, see [Setting the Rate Limit for the Cloud Controller API](./setting-rate-limit-cloud-api.html).
+Note that the different rate limits are independent of each other and therefore multiple rate limit information can be returned.
```

Comment:
I'd tweak this a bit to say something like:

"_Note that the Cloud Controller offers several, distinct rate limiters to serve different purposes. Where multiple rate limiter are enabled, information about each will be shared in its own HTTP response header._"